pocs/observatory.py

```diff
@@ -130,17 +131,23 @@ def primary_camera(self, cam):
 
     @property
     def current_observation(self):
+        if not self.can_observe:
+            raise error.NoObservation("No valid observations found.")
```

Comment:
`can_observe` could evaluate to False for a number of reasons (no scheduler, no cameras, no mount). That is not the same thing as saying there is not a valid observation. This message implies that all of the hardware is correct and that the scheduler is working, it's just that there are no particular targets available at this time, which is not what we want to say.

The specific reason that `can_observe` is False should be handled inside `can_observe` itself.  

Also, one could technically have a `scheduler`, and a `current_observation`, and still `can_observe` is False, e.g. when there are no cameras present.  We shouldn't be prevented from getting the `current_observation` in that situation.

Reply:
According to your commentary, I understood that I should not raise error when can_observe is False. can_observe should display reason of it's falseness ( like no scheduler, no cameras, no mount) and should return True or False. Then, when I will start using it, I am thinking that I can do like that
```if not can_observe:
         return None #what else should I return instead of None or how else I have to stop function if it's not possible to observe```